be/src/common/config.h

```diff
@@ -895,6 +895,7 @@ CONF_String(rocksdb_cf_options_string, "block_based_table_factory={block_cache=1
 // limit local exchange buffer's memory size per driver
 CONF_Int64(local_exchange_buffer_mem_limit_per_driver, "134217728"); // 128MB
 CONF_mInt64(wait_apply_time, "6000");                                // 6s
+CONF_mInt64(wait_for_version_time, "55000"); // 55s, should smaller than exec_plan_fragment rpc timeout(60s)
```

Comment:
It seems the configuration is not necessary. Just setting wait_for_version_time equals to `exec_plan_fragment rpc_timemout - 5` is enough?

Reply:
Yes, it is enough.